core/variables.js

```diff
@@ -270,6 +270,55 @@ Blockly.Variables.createVariable = function(workspace, opt_callback) {
 };
 
 /**
+ * Rename a variable with the given workspace, variableType, and oldName.
+ * @param {!Blockly.Workspace} workspace The workspace on which to rename the
+ *     variable.
+ * @param {function(?string=)=} opt_callback A callback. It will
+ *     be passed an acceptable new variable name, or null if change is to be
+ *     aborted (cancel button), or undefined if an existing variable was chosen.
+ * @param {string} variableType Optional variable type, like 'string' or 'list'.
+ * @param {string} currentName Name of the variable being renamed.
+ */
+Blockly.Variables.renameVariable = function(workspace, opt_callback,
```

Comment:
This is fine, but I think we should come up with a team standard for passing in parameters. Because in another PR, when I passed in variable, I was told to pass in the specific attributes that were used in it rather than the whole variable. Unfortunately I can't find the PR, I just remember the mental note to continue this practice.

Done and done!